scripts/configure-ironic.sh

```diff
@@ -11,16 +11,19 @@ IRONIC_EXTERNAL_IP="${IRONIC_EXTERNAL_IP:-}"
 #   <interface.vlan> - a particular VLAN on an interface, not relying on LLDP
 export IRONIC_INSPECTOR_VLAN_INTERFACES=${IRONIC_INSPECTOR_VLAN_INTERFACES:-all}
 
+# shellcheck disable=SC1091
```

Comment:
you could use curly brackets to group multiple lines and remove the second disable statement, I'm ok either way

Reply:
True. IMO it is easier to understand this way, while being explicit about the ignores.